code/modules/antagonists/paradox_clone/paradox_clone.dm

```diff
@@ -0,0 +1,116 @@
+/datum/antagonist/paradox_clone
+	name = "\improper Paradox Clone"
+	roundend_category = "Paradox Clone"
+	job_rank = ROLE_PARADOX_CLONE
+	prevent_roundtype_conversion = FALSE
+	antag_hud_name = "paradox_clone"
+	suicide_cry = "THERE CAN BE ONLY ONE!!"
+	preview_outfit = /datum/outfit/paradox_clone
+	count_against_dynamic_roll_chance = TRUE
+
+	var/datum/weakref/original_ref ///antags target
+
+/datum/antagonist/paradox_clone/get_preview_icon()
+	var/icon/final_icon = render_preview_outfit(preview_outfit)
+
+	final_icon.Blend(make_background_clone_icon(preview_outfit), ICON_UNDERLAY, -8, 0)
+
+	final_icon.Scale(64, 64)
+
+	return finish_preview_icon(final_icon)
+
+/datum/antagonist/paradox_clone/proc/make_background_clone_icon(datum/outfit/clone_fit)
+	var/mob/living/carbon/human/dummy/consistent/clone = new
+
+	var/icon/clone_icon = render_preview_outfit(clone_fit, clone)
+	clone_icon.ChangeOpacity(0.5)
+	qdel(clone)
+
+	return clone_icon
+
+/datum/outfit/paradox_clone
```

Comment:
outfit should be at the very bottom of the file so it's not mixed up with the antag's code